front/app/containers/Admin/projects/edit/general/index.tsx

```diff
@@ -1,576 +0,0 @@
-        process_type: projectType,
-      },
-    }));
-  };
-
-  handleHeaderOnAdd = (newHeader: UploadFile[]) => {
-    this.setState(({ projectAttributesDiff }) => ({
-      submitState: 'enabled',
-      projectAttributesDiff: {
-        ...projectAttributesDiff,
-        header_bg: newHeader[0].base64,
-      },
-      projectHeaderImage: [newHeader[0]],
-    }));
-  };
-
-  handleHeaderOnRemove = async () => {
-    this.setState(({ projectAttributesDiff }) => ({
-      submitState: 'enabled',
-      projectAttributesDiff: {
-        ...projectAttributesDiff,
-        header_bg: null,
-      },
-      projectHeaderImage: null,
-    }));
-  };
-
-  handleProjectFileOnAdd = (newProjectFile: UploadFile) => {
-    this.setState((prevState) => {
-      const isDuplicate = prevState.projectFiles.some(
-        (file) => file.base64 === newProjectFile.base64
-      );
-      const projectFiles = isDuplicate
-        ? prevState.projectFiles
-        : [...prevState.projectFiles, newProjectFile];
-      const submitState = isDuplicate ? prevState.submitState : 'enabled';
-
-      return {
-        projectFiles,
-        submitState,
-      };
-    });
-  };
-
-  handleProjectFileOnRemove = (projectFileToRemove: UploadFile) => {
-    this.setState(({ projectFiles, projectFilesToRemove }) => ({
-      submitState: 'enabled',
-      projectFiles: projectFiles.filter(
-        (file) => file.base64 !== projectFileToRemove.base64
-      ),
-      projectFilesToRemove: [...projectFilesToRemove, projectFileToRemove],
-    }));
-  };
-
-  handleProjectImagesOnAdd = (projectImages: UploadFile[]) => {
-    this.setState({
-      projectImages,
-      submitState: 'enabled',
-    });
-  };
-
-  handleProjectImageOnRemove = (projectImageToRemove: UploadFile) => {
-    this.setState(({ projectImages, projectImagesToRemove }) => ({
-      submitState: 'enabled',
-      projectImages: projectImages.filter(
-        (image) => image.base64 !== projectImageToRemove.base64
-      ),
-      projectImagesToRemove: [...projectImagesToRemove, projectImageToRemove],
-    }));
-  };
-
-  handleTopicsChange = (topicIds: string[]) => {
-    this.setState(({ projectAttributesDiff }) => ({
-      submitState: 'enabled',
-      projectAttributesDiff: {
-        ...projectAttributesDiff,
-        topic_ids: topicIds,
-      },
-    }));
-  };
-
-  handleAreaTypeChange = (value: 'all' | 'selection') => {
-    this.setState(({ projectAttributesDiff }) => ({
-      submitState: 'enabled',
-      areaType: value,
-      projectAttributesDiff: {
-        ...projectAttributesDiff,
-        area_ids: value === 'all' ? [] : projectAttributesDiff.area_ids,
-      },
-    }));
-  };
-
-  handleAreaSelectionChange = (values: IOption[]) => {
-    this.setState(({ projectAttributesDiff }) => ({
-      submitState: 'enabled',
-      projectAttributesDiff: {
-        ...projectAttributesDiff,
-        area_ids: values.map((value) => value.value),
-      },
-    }));
-  };
```

Comment:
Definitely check them, but these functions have no noteworthy changes except the change to fit the functional component.